terraform/modules/enclave/prometheus/test/fixtures/main.tf

```diff
@@ -0,0 +1,75 @@
+provider "aws" {
+  region = "eu-west-2" # "${var.aws_region}"
+}
+
+terraform {
+  required_version = ">= 0.11.7"
+}
+
+resource "aws_vpc" "main" {
+  cidr_block           = "10.0.0.0/22"
+  enable_dns_hostnames = true
+  enable_dns_support   = true
+}
+
+resource "aws_internet_gateway" "internet_gateway" {
+  vpc_id = "${aws_vpc.main.id}"
+}
+
+module "network" {
+  source              = "../../../../enclave/network"
+  environment         = "perf-test"
```

Comment:
Might be good to call this `verify-perf-a-test` to match the name of the project

Reply:
Also will we run into issues if more than one person tries to run the tests at the same time? (although this probably is something we could survive with for the moment)

Reply:
Yep for the time being it is basic and this means that we can use this until we get a make file. The reason why I avoided that was that I did not want to run in to charecter limits.